src/chi/components/data-table/data-table.scss

```diff
@@ -0,0 +1,721 @@
+        border-bottom: 0.125rem solid $color-border-base-dark;
+        display: flex;
+        font-weight: $font-weight-semi-bold;
+        padding: 0.75rem;
+        position: relative;
+
+        .chi-icon {
+          margin-left: 0.5rem;
+
+          &.icon-arrow-sync {
+            color: $color-icon-muted-light;
+            transform: rotate(90deg);
+            transition: transform 0.2s ease-in-out;
+          }
+        }
+      }
+    }
+  }
+
+  &__body {
+    @include respond-to(md) {
+      display: flex;
+      flex-direction: column;
+    }
+
+    .chi-data-table__cell {
+      display: flex;
+      padding: 0.25rem 0.75rem;
+
+      @include respond-to(md) {
+        border-bottom: 0.0625rem solid $color-border-base-light;
+        padding: 0.75rem;
+        word-break: break-word;
+      }
+
+      .-text--truncate {
+        cursor: pointer;
+      }
+
+      &::before {
+        content: attr(data-label);
+        display: block;
+        font-weight: $font-weight-semi-bold;
+        min-width: 10rem;
+
+        @include respond-to(md) {
+          display: none;
+        }
+      }
+    }
+  }
+
+  &__footer {
+    display: none;
+
+    @include respond-to(md) {
+      background-color: $color-background-white;
+      display: block;
+      padding: 1.5rem 1rem;
+    }
+  }
+
+  &__row {
+    background-color: $color-background-white;
+    border-bottom: 0.0625rem solid $color-border-base-light;
+    display: block;
+    padding: 1rem 0;
+
+    @include respond-to(md) {
+      border: 0;
+      display: flex;
+      flex-direction: row;
+      padding: 0;
+    }
+
+    &.-active {
+      background-color: $color-background-primary-light;
+      box-shadow: inset 0.125rem 0 0 $color-border-primary;
+    }
+
+    &.-selected {
+      background-color: $color-background-primary-light;
+    }
+
+    &.-expanded {
+      background-color: $color-background-primary-light;
+      box-shadow: inset 0.25rem 0 0 $color-border-primary;
+
+      .chi-data-table__cell {
+        &.-expandable {
+          .icon-chevron-right {
+            transform: rotate(90deg);
+            transition: transform 0.2s ease-in-out;
+          }
+        }
+      }
+    }
+
+    &-empty {
+      background-color: $color-background-base;
+      font-style: italic;
+      padding: 0.75rem;
+    }
+
+    &-child,
+    &-grand-child {
+      background-color: $color-background-primary-light;
+      box-shadow: inset 0.25rem 0 0 $color-border-primary;
+
+      @include respond-to(md) {
+        background-color: $color-background-primary-lighter;
+        border-bottom: 0.0625rem solid $color-border-base-light;
+        display: flex;
+        flex-direction: row;
+        padding: 1.5rem 2rem;
+      }
+    }
+  }
+
+  &__cell {
+    @include respond-to(md) {
+      align-items: center;
+      flex-basis: 0%;
+      flex-grow: 1;
+      flex-shrink: 1;
+      justify-content: flex-start;
+    }
+
+    &.-selectable,
+    &.-expandable {
```

Comment:
Can you add `bottom-border-color: transparent;` for expandable cells to eliminate the gray border color that's conflicting with the blue? Should only apply to cells within `chi-data-table__body`, not `chi-data-table__head`.

![image](https://user-images.githubusercontent.com/130654/97207551-63edad80-1777-11eb-85f4-2900e9c2e919.png)